Remotion/Web/Development.WebTesting/Accessibility/AccessibilityAnalyzer.cs

```diff
@@ -225,8 +225,10 @@ private bool AxeIsInjected ()
     /// </remarks>
     private void InjectIntoIFrames (string source)
     {
+      var maximumTimeToWaitForFrame = TimeSpan.FromMilliseconds (3000);
```

Comment:
Okay, I'm afraid this would require us to actually pass the timeout from the public API all the way to down-here. We can define the actual default value once in here, but we should allow injecting a new value.